hera_cal/io.py

```diff
@@ -736,6 +736,8 @@ def get_file_lst_range(filepaths, filetype='uvh5', add_int_buffer=False):
         file_starts : file starting point (bin center) in LST [radians]
         file_stops : file ending point (bin center) in LST [radians]
         int_times : integration duration in LST [radians]
+        file_lst_arrays : ndarrays of file lst_array [radians]
```

Comment:
I think we should modify this function to *only* return these last two. Call it `get_file_times` or something similar. We'll need to modify the function that uses it in abscal, but that's fine.